code/qcommon/ojk_saved_game_helper_fwd.h

```diff
@@ -290,11 +290,8 @@ template<typename T>
 class SavedGameClassArchiver
 {
 public:
-	static constexpr bool is_implemented()
-	{
-		return false;
-	}
-
+	enum { is_implemented = false };
```

Comment:
Same as above.
Valid but looks like a "hack".
Isn't static constant field would be more readable?
(i.e., "static const bool is_implemented = true;")